### ISSUE TITLE
chests.js logic needs to be updated for Open Mode

### ISSUE THREAD

**Issue by crossproduct42** [crossproduct42/alttprandohelper#3]
I see that the Open Mode option automatically leaves the escape chests unchecked, and sets the appropriate default sword value. That's good. The problem is that the logic completely changes in some places: for example, you cannot reach Mothula without a sword. Also, dark rooms follow different rules for Open/Standard modes, so we might want to work that in somehow. (Maybe introduce a blue color that means "You can get this, but it circumvents the item placement logic")

tl;dr
- Certain locations need to include a sword requirement
- Decide whether to consider Lantern as a requirement for dark rooms

**Comment by RebelusQuo** [comment]
Pull request open-mode #2 have been adapted to cover this. I'm closing this issue.
